scripts/HealthCheck.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.digital.ho.hocs;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+/**
+ * Class used for container health check by docker compose
+ */
+public class HealthCheck {
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+        var client = HttpClient.newHttpClient();
+        var request = HttpRequest.newBuilder()
+                .uri(URI.create("http://localhost:8080/actuator/health"))
```

Comment:
Could we use `server.port` to make this more resilient? https://github.com/UKHomeOffice/hocs-casework/blob/c6f7a1ba51a73b716ea057a67855f2d16142349d/src/main/resources/application.properties#L5

Reply:
This file now uses a command line argument to set the end point to check so it is fully customisable